### PR TITLE
fix: type declaration for IncomingPhoneNumberInstance

### LINKED ISSUE #529: Incorrect type declaration for IncomingPhoneNumberInstance
Type of `IncomingPhoneNumberInstance.capabilities` is `string`, but should be `{sms: boolean, voice: boolean, ...}`.

Please, see type definition
https://github.com/twilio/twilio-node/blob/3d0e4a1de19476337f2ac4413b6c612c58b3c721/lib/rest/api/v2010/account/incomingPhoneNumber.d.ts#L398

### PR DESCRIPTION
**Pull request by thinkingserious**
Fixes #529

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: lib/interfaces.d.ts [hunk 1]
```diff
@@ -4,6 +4,12 @@ export type Url = string;
 
 export type PhoneNumber = string;
 
+export type PhoneNumberCapabilities = {
+  mms: boolean;
+  sms: boolean;
+  voice: boolean;
```
Comment (childish-sambino) [inline lib/interfaces.d.ts:10]: What about fax?

Comment (thinkingserious) [reply lib/interfaces.d.ts:10]: I did not see that here:
https://code.hq.twilio.com/twilio/api-definitions/blob/master/transactions/api/v2010/incoming_phone_number.json#L42

Where did you see `fax` defined?

**Comment by thinkingserious** [comment]
Tested with:

```node
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const client = require('./lib')(accountSid, authToken);

client.incomingPhoneNumbers('PNXXXXXXXXXXXXXXXXXXXXXXXXXXX')
    .fetch()
    .then(incoming_phone_number => console.log(incoming_phone_number.capabilities));
```

Response:

`{ voice: true, sms: true, mms: true, fax: true }`

**Comment by childish-sambino** [comment]
@thinkingserious Was that a test using TypeScript?

**Comment by thinkingserious** [comment]
That code was first put into a file `temp.ts` and compiled with `tsc temp.ts` which produced `temp.js`. After executing `node temp.js`, the above output was received.

**Pull request merged**